taxi/admin.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from taxi.models import Manufacturer, Car, Driver
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = ["username", "license_number", ]
+    fieldsets = UserAdmin.fieldsets + (
+        ("Additional info", {"fields": ("license_number",)}),
+    )
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        ("Additional info", {"fields": ("license_number", )}),
+    )
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_filter = ["manufacturer", ]
+    search_fields = ["model", ]
+
+
+admin.site.register(Manufacturer)
```

Comment:
Prefer using one approach for registering your models, either using functions or decorators.